### ISSUE TITLE
Develop Individual Development Plans

### ISSUE THREAD

**Issue by its-a-lisa** [18F/tts-tech-portfolio#238]
## User Story:
As someone working in the Tech Portfolio, I would like to have an Individual Development Plan
## Problem Statement: 
 Without an IDP it is difficult to justify training, conferences and developmental activities.
## Background information: 
I've been asked to go on podcasts and there have been announcements for 'highly attended events' that I've wanted to submit for
## Actions to take:  
- [ ] Pre-Planning - supervisor and employee prepare independently for meeting
Employee/Supervisor Meeting - discuss employee strengths, areas for improvement, interests, goals, and organizational requirements
- [ ] Prepare IDP - employee, in consultation with supervisor, completes plan for individual development
  - [ ] Training Requests
  - [ ] Conference Requests
  - [ ] Development Requests
## Acceptance criteria: 
- [ ] IDP is accept and approved
## Open New Issue with following actions: 
- [ ] Implement Plan - employee pursues training and development identified in plan
- [ ] Evaluate Outcomes - supervisor/employee evaluate usefulness of training and development experiences

## Related Issues: 
https://github.com/18F/tts-tech-portfolio/issues/171
https://github.com/18F/tts-tech-portfolio/issues/166
##Blockers:
https://github.com/18F/tts-tech-portfolio/issues/171

**Comment by its-a-lisa** [comment]
As a member of the Tech Portfolio I would like training and practice on developing user stories.

**Comment by its-a-lisa** [comment]
Azure Fundamentals?

**Comment by JJediny** [comment]
Blocked by https://github.com/18F/tts-tech-portfolio/issues/278 ?

**Comment by afeld** [comment]
Perhaps ideally, but given how long that's been taking, I wouldn't wait for it. Can always tweak after the fact, though I wouldn't expect the Performance Plans to cause major changes to the Development Plans.
